website/blog/2024-04-22-extended-attributes.md

```diff
@@ -0,0 +1,103 @@
+
+All you need to do is configure an Environment as Staging, and toggle the control in the dbt Cloud IDE to “Defer to staging/production”. This will favor a Staging environment over Prod if you have one set up.
+
+![Configure environment as Staging](/img/blog/2024-04-10-extended-attributes/env_settings.png)
+_<small>You can now designate a Staging Environment in a first-class way.</small>_
+
+![Defer to staging/production](/img/blog/2024-04-10-extended-attributes/defer_to_stage.png)
+_<small>Make sure to toggle on 'Defer to staging/production' to save time and money!</small>_
+
+## Upgrading on a curve
+
+Lastly let’s consider a more specialized use case. Imagine we have a ‘tiger team’ (consisting of a lone analytics engineer named Dave) tasked with upgrading from dbt version 1.6 to the new ‘Keep on latest version’ setting, to take advantage of added stability and feature access. We want to keep the rest of the data team being productive in dbt 1.6 for the time being, while enabling Dave to upgrade and do his work in the new versionless mode.
+
+### The Development Environment
+
+![Development Environment configured to be 1.6 by default](/img/blog/2024-04-10-extended-attributes/dbt_version.png)
+
+The Development Environment is configured to be 1.6 by default.
+
+### Dave’s Development connection settings
+
+![Dave's development version override](/img/blog/2024-04-10-extended-attributes/dave_version.png)
+
+## Launch Special
+
+Each connection you make from every environment is now unique. You can deploy, develop, and test your data with a setup that molds to your organization, not to what’s available in dbt Cloud. Whether you’re looking to create advanced, layered environments to launch new models safely, or enable greater independence between developers, dbt Cloud extends to support what you need. The best part is, we're just getting started: the upcoming Global Connections feature set will take this even further, allowing you to set multiple connections globally and reuse them wherever needed.
+
+I encourage you to take these new features for a spin by creating a Staging environment, configuring the unique connections you need to enable it at your org, and seeing how it can make your data team more efficient and secure. As always, if you need help or have questions, the [dbt Community Forum](https://discourse.getdbt.com/) and [Slack](https://www.getdbt.com/community/join-the-community) are here to support you. Happy modeling!
```

Comment:
```suggestion
I encourage you to take these new features for a spin by creating a staging environment, configuring the unique connections you need to enable it at your org, and seeing how it can make your data team more efficient and secure. As always, if you need help or have questions, the [dbt Community Forum](https://discourse.getdbt.com/) and [Slack](https://www.getdbt.com/community/join-the-community) are here to support you. Happy modeling!
```